Website/Packages/datax-pipeline/src/modules/flowDefinition/flowActions.js

```diff
@@ -93,6 +95,12 @@ export const initFlow = context => (dispatch, getState) => {
             })
             .then(flow => {
                 dispatch(QueryActions.initQuery(flow.payload.query));
+                isDatabricksSparkType().then(response => {
+                    return dispatch({
```

Comment:
Let's create a function like this and use it.
function updateIsDatabricksSparkType(dispatch, value) {
    return dispatch({
        type: 'FLOW_UPDATE_ISDATABRICKSSPARKTYPE',
        payload: response
    });
}

Reply:
Done